compiler/test/compilable/test20039.d

```diff
@@ -1,5 +1,5 @@
 void foo()() { }
-class bar { }
+void bar(int) { }
```

Comment:
The reason this is changed is that now an error would be emitted. However I'ce checked that the issue (ICE) was also reproducible with a function (https://godbolt.org/z/Y6EqqGcac)